src/sys/socket/sockopt.rs

```diff
@@ -368,7 +368,7 @@ impl<'a> Set<'a, usize> for SetUsize {
     }
 
     unsafe fn ffi_ptr(&self) -> *const c_void {
```

Comment:
That would be required, yes. Could you change those in a separate commit after the current changes? I'm pretty certain the other function in `Get` are also safe, but I don't feel like getting familiar with it enough here to suggest we change those as well.